scipy/stats/_continuous_distns.py

```diff
@@ -6411,6 +6411,13 @@ def _stats(self, nu):
         g2 /= nu*mu2**2.0
         return mu, mu2, g1, g2
 
+    def _entropy(self, nu):
```

Comment:
Intermediate calculations will over/underflow for moderately large and small `nu`, making this formulation unsuitable for very large and very small `nu`. You can use the fact that `log(A * B * C) = log(A) + log(B) + log(C)` to prevent this.

Once you do that, you'll find the magnitude of terms `logA` and `logC` are large and of opposite sign. They should cancel out, leaving just term `logB`, but that's not what will happen in some formulations.  

Can you fix these things so that this is accurate for both extremely large and small `nu` (e.g. `nu = 1e-100`, `nu=1e100`) before review continues?

Reply:
Done.

Reply:
With the first paragraph, but not the rest. 
Please see https://github.com/scipy/scipy/pull/17873/files#r1089876313 since we still have the same problem in both places.
Again, please let's finish one of these first so we don't need to double everything.